prow/cmd/deck/main.go

```diff
@@ -784,6 +784,12 @@ func renderSpyglass(sg *spyglass.Spyglass, cfg config.Getter, src string, o opti
 		jobHistLink = path.Join("/job-history", jobPath)
 	}
 
+	prowJobLink := ""
+	prowJobPath, err := sg.ProwJobPath(src)
+	if err == nil {
```

Comment:
Please log this error at the warning level. (But don't log 'prowjob not found' errors. See the comment about handling such errors differently.)